jobs/port_binary_distributable.tcl

```diff
@@ -284,7 +283,7 @@ proc check_licenses {portName variantInfo verbose} {
 
 proc split_variants {variants} {
     set result {}
-    set l [regexp -all -inline -- {([-+])([[:alpha:]_]+[\w\.]*)} $variants]
+    set l [regexp -all -inline -- {(?:[[:space:]]|^)([-+])([[:alpha:]_]+[\w\.]*)(?:[[:space:]]|$)} $variants]
```

Comment:
Not sure whether I understand this change. Why would it be necessary to match the spaces explicitly now, but was not before?